### PR TITLE
feat: created my profile side panel

### PR DESCRIPTION
**Pull request by mgaseta**
# Description

Created the structure for right side panels and finished My Profile section.
[JIRA Ticket](https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-328)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Manually tested only.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

### CODE REVIEW

File: src/components/MyProfile/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  return (
+    <>
+      <div className="user-info-section">
+        <div className="user-image">
+          <AvatarImage userName={UserData.name} size="large" />
+        </div>
+        <div className="user-data">
+          <p className="user-name">{UserData.name}</p>
+          <p>{`IDIR: ${UserData.idir}`}</p>
+          <p>{UserData.email}</p>
```
Comment (RMCampos) [inline src/components/MyProfile/index.tsx:37]: I think we can replace these mocked data with real data from the logged user.

**Review by RMCampos — CHANGES_REQUESTED** [review]
Great job!! 🤩

File: src/components/MyProfile/index.tsx (hunk 1 above)
Comment (mgaseta) [reply src/components/MyProfile/index.tsx:37]: Yes, we can. I'll adjust that 🐱‍🏍.

**Review by RMCampos — APPROVED** [review]
Great job!! 🥳

**Pull request merged**